diagnostic.md

```diff
@@ -36,13 +39,16 @@ Now, we want to get a **single donut** from the server. Write the `curl` request
 you'd use to retrieve a **single donut**, using whatever ID you'd like.
 
 ```sh
-# your answer here
+# curl --include --request GET http://www.example.com/donut
 ```
 
 Write an AJAX request to retrieve a **single donut** from the server.
 
 ```js
-let getDonut = /* your answer here */;
+let getDonut = $.ajax({
+  url: config.apiOrigin + '/donut',
```

Comment:
This will only work if `config.apiOrigin` exists and is in the correct format and the full url needs to be similar to the one above.

```javascript
let getDonut = function (donutID) {
  return $.ajax({
    url: 'http://www.example.com/donuts/' + donutID,
    method: 'GET',
  });
};
```